tests/semantic_analyser.cpp

```diff
@@ -3757,6 +3757,16 @@ BEGIN { @map[0] = 1; for ($kv : @map) { print($kv); } }
              false);
 }
 
+TEST(semantic_analyser, for_loop_no_ctx_access)
+{
+  test_error("kprobe:f { @map[0] = 1; for ($kv : @map) { arg0 } }",
+             R"(
+stdin:1:45-49: ERROR: 'arg0' builtin is not allowed in a for-loop
+kprobe:f { @map[0] = 1; for ($kv : @map) { arg0 } }
+                                            ~~~~
```

Comment:
Is it a known bug that the spacing is off here?

Reply:
Yeah, I noticed, too. I've never realized before so it may be an unknown bug.

Reply:
Ah here it is: https://github.com/bpftrace/bpftrace/issues/3063